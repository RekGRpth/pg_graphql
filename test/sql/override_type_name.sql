begin;
    create table account(
        id serial primary key,
        email varchar(255) not null
    );

    comment on table public.account is E'@graphql({"name": "UserAccount"})';

    select graphql.rebuild_schema();

    select name from graphql.type where entity = 'public.account'::regclass order by name;

rollback;
