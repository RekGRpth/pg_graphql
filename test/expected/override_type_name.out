begin;
    create table account(
        id serial primary key,
        email varchar(255) not null
    );
    comment on table public.account is E'@graphql({"name": "UserAccount"})';
    select name from graphql.type where entity = 'public.account'::regclass order by name;
         name          
-----------------------
 UserAccount
 UserAccountConnection
 UserAccountEdge
 UserAccountFilter
 UserAccountOrderBy
(5 rows)

rollback;
